Cargo.lock

```diff
@@ -3960,7 +3960,7 @@ dependencies = [
  "pallet-utility",
  "pallet-vesting",
  "pallet-xcm",
- "pallet-xcm-benchmarks",
+ "pallet-xcm-benchmarks 0.9.27 (git+https://github.com/paritytech/polkadot?branch=master)",
```

Comment:
?

Reply:
This is to distinguish it from the other crate in polkadot.

Reply:
https://github.com/paritytech/cumulus/issues/1569